frontend/ui/screensaver.lua

```diff
@@ -385,7 +388,7 @@ function Screensaver:setup(event, fallback_message)
     if G_reader_settings:has(self.prefix .. "screensaver_type") then
         self.screensaver_type = G_reader_settings:readSetting(self.prefix .. "screensaver_type")
     else
-        if event then
+        if event and not G_reader_settings:isTrue("screensaver_disable_fallbackmsg") then
```

Comment:
`isFalse` is slightly less tortuous ^^.

(And being able to use it here, and `toggle` in the Settings, is actually the reasoning behind the forced initialization ;)).

Reply:
I'd probably keep a `_` between `fallback` and `msg` in the setting key, while you're there ;).

Reply:
Yeah, I was confused about the convention. Some settings have `msg` others have `message` spelled out. And none seem to exceed 3 words.

Reply:
Yeah, that was more coincidence and things growing out in a slightly haphazard fashion that anything.

I'd KISS: a sensible prefix, and a meaningful name. We don't really care about length ;).

Reply:
Just realized that on devices with poor power management, it makes more sense to have a message displayed when sleeping (because of the battery drain) and not when powered off. In other words, the opposite of a normal ereader. Is there a not convoluted way of adding that to this PR?

Reply:
I would probably no rock the boat too much and leave that one to user preferences (by, e.g., setting a custom message)?

If you suffered through the surrounding code, you'll maybe have noticed that those options support custom variants prefixed by those event names, so you can technically get a custom message that's different for sleep, reboot & poweroff.

Exposing that in the UI would be a bit nightmarish, hence the current status quo ^

Reply:
Fair enough. Sounds like that would be a more substantial overhaul. Let's leave this PR like this then.